plugins/UM3NetworkPrinting/resources/qml/MonitorConfigOverrideDialog.qml

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.3
+import QtQuick.Controls 1.4
+import QtQuick.Layouts 1.3
+import QtQuick.Dialogs 1.2
+import UM 1.3 as UM
+
+UM.Dialog
+{
+    id: overrideConfirmationDialog
+
+    property var printer: null
+
+    minimumWidth: screenScaleFactor * 640;
+    minimumHeight: screenScaleFactor * 320;
```

Comment:
It is hard to kick the semi-colon habit, I know.

Reply:
Actually this was copy pasta'd from somewhere else because... you know... we totes use ; all over Cura 🙄